.github/workflows/e2e:install-ui.yml

```diff
@@ -38,8 +38,13 @@ jobs:
         run: |
           sudo systemctl start mysql
           mysql -uroot -proot -e 'CREATE DATABASE flarum_test;' --port 13306
-
+      
+      
       - name: Cypress run
         uses: cypress-io/github-action@v1
         with:
-          config-file: cypress-install.json
\ No newline at end of file
+          config-file: cypress-full.json
```

Comment:
Why was the config file for this one changed? We have 2 parallel testing processes, one for the installation GUI, one for the main site.

Reply:
Yeah @katosdev this should still be `cypress-install.json`

Reply:
Whoops - rookie copy/paste issue, sorry guys! @datitisev @askvortsov1 
Good spot :)